accounts/accounts.go

```diff
@@ -237,11 +237,12 @@ func (aS *AccountS) accountDebit(ctx *context.Context, acnt *utils.Account, usag
 // V1AccountsForEvent returns the matching Accounts for Event
 func (aS *AccountS) V1AccountsForEvent(ctx *context.Context, args *utils.CGREvent, aps *[]*utils.Account) (err error) {
 	var accIDs []string
-	accIDs = aS.cfg.AccountSCfg().Opts.AccountIDs
-	if args.APIOpts[utils.OptsAccountsAccountIDs] != nil {
-		if accIDs, err = utils.IfaceAsStringSlice(args.APIOpts[utils.OptsAccountsAccountIDs]); err != nil {
-			return
-		}
+	if accIDs, err = engine.FilterStringSliceCfgOpts(ctx, args.Tenant, args.AsDataProvider(), aS.fltrS,
```

Comment:
Need to discuss about the name starting with Filter since I am not sure I get the purpose of the function.